docs/releasing.md

```diff
@@ -66,11 +66,16 @@ git push -f origin stable
 
 If this is GA:
 
-* [ ] Update the [Homebrew tap](https://github.com/argoproj/homebrew-tap).
-* [ ] Check Homebrew was successfully updated:
+* [ ] Update the Homebrew formula.
+
+```bash
+brew bump-formula-pr argo --version $VERSION
+```
+
+* [ ] Check that Homebrew was successfully updated after the PR was merged:
  
  ```
- brew upgrade argoproj/tap/argo
+ brew upgrade argo
```

Comment:
Would the old way `brew upgrade argoproj/tap/argo` still work?

Reply:
@terrytangyuan I added a `tag_migrations.json` to your homebrew-tap here https://github.com/argoproj/homebrew-tap/pull/15
When calling `brew upgrade argoproj/tap/argo` the user would get this:
It automatically uses the new tap for installation `homebrew-core`

```
brew upgrade argoproj/tap/argocd
Warning: Use argocd instead of deprecated argoproj/tap/argocd
Warning: Use argocd instead of deprecated argoproj/tap/argocd
==> Upgrading 1 outdated package:
argocd v1.6.0 -> 1.6.1
==> Upgrading argocd v1.6.0 -> 1.6.1
```

Reply:
Sounds good. Thanks!